config/database.yml

```diff
@@ -18,11 +18,11 @@ default: &default
 development:
```

Comment:
Did you intend to push this database.yml config too?

Reply:
No, of course not. I restore the database.yml.